deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml

```diff
@@ -55,83 +82,23 @@ spec:
       kind: SmartGateway
       name: smartgateways.smartgateway.infra.watch
       resources:
-      - kind: Pod
-        name: ""
-        version: v1
-      - kind: Deployments
-        name: ""
-        version: v1
-      - kind: ReplicaSets
-        name: ""
-        version: v1
       - kind: Services
         name: ""
         version: v1
-      - kind: ConfigMaps
-        name: ""
-        version: v1
       - kind: SmartGateways
         name: smartgateways.smartgateway.infra.watch
-        version: v2alpha1
+        version: v2
       - kind: ServiceMonitors
         name: servicemonitors.monitoring.coreos.com
         version: v1
-      specDescriptors:
-      - description: Number of Smart Gateways to deploy
-        displayName: Size
-        path: size
-        x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:number
-      - description: Location of the AMQP endpoint to connect the Smart Gateway to
-        displayName: AMQP URL
-        path: amqpUrl
-        x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:text
-      - description: Container image path
-        displayName: Container image path
-        path: containerImagePath
-        x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:text
-      - description: Number of messages that we can prefetch from AMQP 1.x. By enabling
-          prefetching, the smart gateway won't need to request every message individually
-          from the AMQP bus, resulting in a round trip for every request between sender
-          and receiver. To avoid the round trip for every message, the use of prefetch
-          can be used to allow the receiver to request messages be sent in anticipation
-          of them being sent to us.
-        displayName: Prefetch
-        path: prefetch
-        x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:number
-      - description: Use the source timestamp (time when data was collected) rather
-          than let Prometheus write when the data was scraped for that metric.
-        displayName: Use Timestamp
-        path: useTimestamp
-        x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
-      - description: Smart Gateway Service Type
-        displayName: Service Type
-        path: serviceType
-        x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:select:metrics
-        - urn:alm:descriptor:com.tectonic.ui:select:events
-      - description: Enable additional debugging information to console output
-        displayName: Enable debugging
-        path: debug
-        x-descriptors:
-        - urn:alm:descriptor:tectonic.ui:booleanSwitch
-      statusDescriptors:
-      - description: Conditions provided by deployment
-        displayName: Conditions
-        path: conditions
-        x-descriptors:
-        - urn:alm:descriptor:io.kubernetes:conditions
-      version: v2alpha1
```

Comment:
Dropped all the `specDescriptors` since the example CR in the CSV is used to populate the OpenShift UI when creating an object. We could go back through this in the future if we want to better construct the UI but since this is generally not an admin facing interface (Service Telemetry Operator is) it's not necessary right now.

If we decide to start making use of Smart Gateway Operator without it being fronted by Service Telemetry Operator we can revisit this.

Reply:
ACK that this is a minor step backwards,  but there is unlikely to be anyone out there using the SGO directly while also relying on the UI hints. It's been a long while since I've looked at the it.